lua/modules/configs/completion/formatting.lua

```diff
@@ -105,7 +84,7 @@ function M.format_filter(clients)
 		end)
 		if status_ok and formatting_supported and client.name == "null-ls" then
 			return "null-ls"
-		elseif client.name ~= "lua_ls" and client.name ~= "tsserver" and client.name ~= "clangd" then
+		elseif not require("core.settings").server_formatting_block_list[client.name] then
 			return status_ok and formatting_supported and client.name
```

Comment:
```suggestion
		elseif not require("core.settings").server_formatting_block_list[client.name] and status_ok and formatting_supported then
			return client.name
```

minor cleanup @Jint-lzxy